operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-secrets-config.yaml

```diff
@@ -50,6 +50,7 @@ metadata:
   namespace: openshift-pipelines
   annotations:
     argocd.argoproj.io/sync-wave: "1"
+    argocd.argoproj.io/hook-delete-policy: BeforeHookCreation
```

Comment:
a comment explaining this argo feature so someone later on does not have to look it up would be nice

Reply:
done